pipelines/deployer/deployer.yaml

```diff
@@ -1140,6 +1140,26 @@ jobs:
     image: task-toolbox
     params:
       CONFIG_VALUES_PATH: ((config-values-path))
+  - task: final-cleanup
```

Comment:
I think this task needs to be the very last thing in the destroy job ... you can't clean up the left over things before the rest have been destroyed

Reply:
moved as requested